doppler/assets/logs/doppler.yaml

```diff
@@ -0,0 +1,56 @@
+id: doppler
+metric_id: doppler
+backend_only: false
+facets:
+  - groups:
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+  - groups:
+      - User
+    name: User Email
+    path: usr.email
+    source: log
+  - groups:
+      - Event
+    name: Type
+    path: type
+    source: log
+pipeline:
+  type: pipeline
+  name: Doppler
+  enabled: true
+  filter:
+    query: "source:doppler"
+  processors:
+    - type: date-remapper
+      name: Define `createdAt` as the official date of the log
+      enabled: true
+      sources:
+        - createdAt
+    - type: message-remapper
+      name: Define `text` as the official message of the log
+      enabled: true
+      sources:
+        - text
```

Comment:
rechecking at the samples, I don't see any sample with this text property. Should all samples cover this property or could you provide at least one with this

Reply:
Great callout, I think I accidentally removed those when I reformatted the logs. I've added them back